io.github.alamahant.Asteria.appdata.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.alamahant.Asteria</id>
+  <name>Asteria</name>
+  <summary>Astrological chart calculator and analyzer with AI interpretations</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>
+      Asteria is a comprehensive astrological chart application that combines traditional astrology with modern AI technology.
+    </p>
+    <p>
+      Features:
+    </p>
+    <ul>
+      <li>Calculate and display natal charts with precise planetary positions</li>
+      <li>Analyze aspects between planets with detailed interpretations</li>
+      <li>View planetary positions by house and sign</li>
+      <li>Examine element and modality distributions</li>
+      <li>Generate AI-powered chart interpretations for deeper insights</li>
+      <li>Calculate transit charts to see how current planetary positions affect your natal chart</li>
+      <li>Clean, intuitive interface for both beginners and experienced astrologers</li>
+    </ul>
+  </description>
+  
+  <url type="homepage">https://github.com/alamahant/Asteria</url>
+  <url type="bugtracker">https://github.com/alamahant/Asteria/issues</url>
+  
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main chart view</caption>
+      <image>https://raw.githubusercontent.com/alamahant/Asteria/main/screenshots/Screenshot_2025-04-21_18-58-06.png</image>
```

Comment:
screenshot should NEVER point to the tip of a branch as it's not immutable.
(this applies to all of them)